scripts/formatter.sh

```
@@ -1,12 +1,23 @@
+
+find \
     mrpt_generic_sensor \
     mrpt_sensor_bumblebee_stereo \
     mrpt_sensor_gnss_nmea \
     mrpt_sensor_gnss_novatel \
     mrpt_sensor_imu_taobotics \
     mrpt_sensorlib \
     mrpt_sensors \
-    -iname *.h -o -iname *.hpp -o -iname *.cpp -o -iname *.c | xargs clang-format-14 -i
+    -iname "*.h" -o -iname "*.hpp" -o -iname "*.cpp" -o -iname "*.c" \
+  -print0 | xargs -0 clang-format-14 "${MODE[@]}"
```

Shellcheck (0.11.0)

[warning] This action ignores everything before the -o. Use \( \) to group.

(SC2146)